### PR TITLE
feat(new-trace): Mimicing issues landing list truncating behaviour for drawer

### PR DESCRIPTION
**Pull request by Abdkhan14**
Minimum width for vertical layout:
Screenshot 2024-04-02 at 9 06 48 PM

### CODE REVIEW

File: static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx [hunk 1]
```diff
@@ -264,9 +267,18 @@ const UsersHeading = styled(Heading)`
   justify-content: center;
 `;
 
-const StyledPanel = styled(Panel)`
+const StyledPanel = styled(Panel)<{width?: number}>`
   margin-bottom: 0;
   border: 1px solid ${p => p.theme.red200};
+
```
Comment (JonasBa) [inline static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx:273]: I would just use a container query here.

File: static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx [hunk 2]
```diff
@@ -123,12 +125,13 @@ type IssueListProps = {
...
 
 export function IssueList({issues, node, organization}: IssueListProps) {
+  const panelRef = useRef<HTMLDivElement>(null);
   if (!issues.length) {
     return null;
   }
 
   return (
-    <StyledPanel>
+    <StyledPanel ref={panelRef} width={panelRef.current?.clientWidth}>
```
Comment (JonasBa) [inline static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx:134]: You need to ensure this doesnt conflict with the resizing behavior as we would now have two different min width definitions. I would use a container query here

**Review by JonasBa — CHANGES_REQUESTED** [review]

**Review by JonasBa — APPROVED** [review]

File: static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx (hunk 2 above)
Comment (JonasBa) [inline static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx:134]: Can you just remove this now? We dont need it anymore

**Comment by JonasBa** [comment]
@Abdkhan14 I approved it, but please just remove the ref and width const

File: static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx (hunk 2 above)
Comment (Abdkhan14) [reply static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx:134]: Woops yup, that shouldn't be there anymore.

**Pull request merged**